src/main.cpp

```diff
@@ -25,6 +23,7 @@
 #include "GlobalNamespace/MenuTransitionsHelper.hpp"
 #include "Zenject/DiContainer.hpp"
 #include "System/Action_1.hpp"
+#include "static-defines.h"
```

Comment:
ties into "static defines file not needed" but this should not be here

Reply:
As above, mod won't compile without this because of references to ID